src/hooks/useIsWindowVisibleRef.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useRef } from 'react'
+
+const VISIBILITY_STATE_SUPPORTED = 'visibilityState' in document
+
+// Check if the tab is active in the user browser
+export default function useIsWindowVisibleRef() {
+  const isBrowserTabActiveRef = useRef(true)
```

Comment:
This is not needed. A boolean is a primitive value and there is no point in wrapping it in a `useRef`